lua/nvim-tree/explorer/explore.lua

```diff
@@ -35,6 +35,7 @@ local function populate_children(handle, cwd, node, git_status, parent)
     end
 
     local abs = utils.path_join { cwd, name }
+    t = t or (vim.loop.fs_stat(abs) or {}).type
```

Comment:
This is an extra (not expensive as usually short circuited) call to fs_stat.

Could we put this after line 44?

Reply:
Moved after line 44, please review.